kinto/tests/core/test_initialization.py

```diff
@@ -166,74 +152,74 @@ def test_does_raise_valueerror_if_entries_are_not_hashable(self):
     def test_raises_valueerror_if_different_multiple_entries(self):
         settings = {
             'paginate_by': 42,
-            'cliquet.paginate_by': 3.14,
+            'kinto.paginate_by': 3.14,
         }
         with self.assertRaises(ValueError):
             self.settings(settings)
 
-        settings = {
-            'kinto.paginate_by': 42,
-            'cliquet.paginate_by': 3.14,
-        }
-        with self.assertRaises(ValueError):
-            self.settings(settings)
+        # settings = {
+        #     'kinto.paginate_by': 42,
+        #     'kinto.paginate_by': 3.14,
+        # }
+        # with self.assertRaises(ValueError):
+        #     self.settings(settings)
```

Comment:
I left this in because I didn't know what I was going to do with the settings prefix stuff. I've removed them from public settings, but haven't taken them out of this yet. I'm going to delete this particular block of code but leave the remainder of the test, which checks that `kinto.setting` is the same as `setting`. Good catch!